website/content/en/docs/deployment/vanilla/guide.md

```diff
@@ -33,290 +25,44 @@ Option 2 targets customization and ability to pick and choose individual compone
-
-If you run all of the following commands, the end result is the same as installing everything through the [single command installation](#install-with-a-single-command). 
-
-The purpose of this section is to:
-- Provide a description of each component and insight on how it gets installed.
-- Enable the user or distribution owner to pick and choose only the components they need.
-
-#### cert-manager
-
-`cert-manager` is used by many Kubeflow components to provide certificates for
-admission webhooks.
-
-Install `cert-manager`:
-
-```sh
-kustomize build upstream/common/cert-manager/cert-manager/base | kubectl apply -f -
-kustomize build upstream/common/cert-manager/kubeflow-issuer/base | kubectl apply -f -
-```
-
-#### Istio
-
-Istio is used by many Kubeflow components to secure their traffic, enforce
-network authorization, and implement routing policies.
-
-Install Istio:
-
-```sh
-kustomize build upstream/common/istio-1-14/istio-crds/base | kubectl apply -f -
-kustomize build upstream/common/istio-1-14/istio-namespace/base | kubectl apply -f -
-kustomize build awsconfigs/common/istio | kubectl apply -f -
-```
-
-#### Dex
-
-Dex is an OpenID Connect Identity (OIDC) with multiple authentication backends. In this default installation, it includes a static user with the email `user@example.com`. By default, the user's password is `12341234`. For any production Kubeflow deployment, you should change the default password by following the steps in [Change default user password](#change-default-user-password).
-
-Install Dex:
-
-```sh
-kustomize build upstream/common/dex/overlays/istio | kubectl apply -f -
-```
-
-#### OIDC AuthService
-
-The OIDC AuthService extends your Istio Ingress-Gateway capabilities to be able to function as an OIDC client:
-
-Install OIDC AuthService:
+Install all Kubeflow official components (residing under `apps`) and all common services (residing under `common`) using either Kustomize or Helm with a single command:
 
-```sh
-kustomize build upstream/common/oidc-authservice/base | kubectl apply -f -
-```
-
-#### Knative
-
-Knative is used by the KServe official Kubeflow component.
-
-Install Knative Serving:
-
-```sh
-kustomize build upstream/common/knative/knative-serving/overlays/gateways | kubectl apply -f -
-kustomize build upstream/common/istio-1-14/cluster-local-gateway/base | kubectl apply -f -
-```
-
-Optionally, you can install Knative Eventing, which can be used for inference request logging.
-
-Install Knative Eventing:
-
-```sh
-kustomize build upstream/common/knative/knative-eventing/base | kubectl apply -f -
-```
-
-#### Kubeflow namespace
-
-Create the namespace where the Kubeflow components will live. This namespace
-is named `kubeflow`.
-
-Install the `kubeflow` namespace:
-
-```sh
-kustomize build upstream/common/kubeflow-namespace/base | kubectl apply -f -
-```
+{{< tabpane persistLang=false >}}
+{{< tab header="Kustomize" lang="toml" >}}
+make deploy-kf-vanilla
```

Comment:
I believe there is no make option named deploy-kf-vanilla

Reply:
also lets be explicit in specifying the INSTALLATION_OPTION=kustomize and DEPLOYMENT_OPTION=vanilla

We should export them before instead of specifying them with the command because for some other options, it might be lots of variables